app/views/catalog/_stcat_tree_show.html.haml

```diff
@@ -1,15 +1,14 @@
 .maincontent
   = render :partial => "layouts/flash_msg"
-  %table
-    %tr
-      %td{:valign => "center"}
-        %table.style1
-          %tr
-            %td.key=_('Name')
-            %td.notwide= @record.name
-          %tr
-            %td.key=_('Description')
-            %td.notwide= @record.description
+  %form.form-horizontal
```

Comment:
@PanSpagetka add static class "form.form-horizontal.static" - this is needed when there are no form inputs (display only)